spec/controllers/videos_controller_spec.rb

```diff
@@ -3,59 +3,59 @@
 describe VideosController do
   
   describe "user access" do
-  before :each do
-    user = User.create(username:"fake person", email: "facke@fake.com", password: "fakepassword")
-    session[:user_id] = user.id
-  end
+    before :each do
+      user = User.create(username:"fake person", email: "facke@fake.com", password: "fakepassword")
+      session[:user_id] = user.id
+    end
   
```

Comment:
the new indentation is correct. However, what about doing something similar to `set_current_admin`?